example/spec/helpers/apiUtils.js

```diff
@@ -12,6 +12,24 @@ function setDistributionApiEnvVars() {
   }
 }
 
+async function waitForApiStatus(getMethod, params, status, config) {
+  return await pRetry(
+    async () => {
+      const record = await getMethod(params);
+
+      if (record.status !== status) {
+        console.log((`Record status ${record.status}. Expect status ${status}`));
```

Comment:
Should we use logger here? Or is this not meant to be here?

Reply:
Oops, debugging log, will remove 😅